### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by frk-dc**
Update dependencies

### CODE REVIEW

File: azure-devops-extension/run-automator-build-task/package.json [hunk 1]
```diff
@@ -16,46 +16,49 @@
...
+    "@types/node-fetch": "^2.6.2",
     "@types/q": "^1.5.4",
     "@types/sinon": "^9.0.8",
-    "@typescript-eslint/eslint-plugin": "^4.1.1",
-    "@zerollup/ts-transform-paths": "^1.7.18",
-    "eslint": "^7.9.0",
-    "eslint-config-deepcrawl": "^5.6.0",
+    "@typescript-eslint/eslint-plugin": "^5.54.1",
+    "eslint": "^8.35.0",
+    "eslint-config-deepcrawl": "^8.1.0",
```
Comment (AndKiel) [inline azure-devops-extension/run-automator-build-task/package.json:31]: After upgrade you should remove unnecessary dependencies, we simplified config to include plugins: https://github.com/deepcrawl/eslint-config/blob/2f855b7749948531a73c6624f64babdeaf5db256/package.json#L23

Comment (frk-dc) [reply azure-devops-extension/run-automator-build-task/package.json:30]: Redundant eslint plugin dependencies removed

**Review by AndKiel — DISMISSED** [review]

**Comment by AndKiel** [comment]
What's the reason for replacing pino with winston? pino logger is our logger of choice in API team (not sure about others but I think they use pino too).

**Comment by frk-dc** [comment]
> What's the reason for replacing pino with winston?

`azure-pipelines-task-lib` no longer supports file names with whitespaces, `pino-pretty` has a peer dependency on a module that's published with a test file that breaks that rule ([help-me](https://github.com/mcollina/help-me/blob/master/fixture/sameprefix/hello%20world.txt)).
Replacing the logger and matching the format was preferred to keeping an outdated `azure-pipelines-task-lib`.

**Review by AndKiel — APPROVED** [review]

**Review by Nikamura — APPROVED** [review]

File: azure-devops-extension/vss-extension.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "automator-tools",
   "name": "Protect Tools",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (Nikamura) [inline azure-devops-extension/vss-extension.json:5]: it's already published as `1.1.4`?

**Pull request merged**